lnbits/decorators.py

```diff
@@ -132,8 +134,9 @@ async def require_invoice_key(
 async def check_access_token(
     header_access_token: Annotated[Union[str, None], Depends(oauth2_scheme)],
     cookie_access_token: Annotated[Union[str, None], Cookie()] = None,
+    bearer_access_token: Annotated[Union[str, None], Depends(http_bearer)] = None,
 ) -> Optional[str]:
-    return header_access_token or cookie_access_token
+    return header_access_token or cookie_access_token or bearer_access_token
```

Comment:
not quite sure if those are even need, or the bearer is handled elsewhere already. @motorina0

Reply:
ok, without this it wont show up in swagger, maybe now there is some dead code around aswell?